transport/http/do_test.go

```diff
@@ -0,0 +1,164 @@
+package http
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/yarpc/internal/testtime"
+	"bytes"
+)
+
+
+func TestCallHttpSuccess(t *testing.T) {
```

Comment:
Since this client can express method and path, we should assert that these get communicated.